src/Tools/Source/RunTests/ConsoleUtil.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using RunTestsUtils;
```

Comment:
remvoe unnecessary using?

Reply:
`Logger` is in RunTestsUtils (shared file)